src/chain/tezos/TezosNodeWriter.ts

```diff
@@ -376,133 +364,104 @@ export namespace TezosNodeWriter {
      */
     export async function sendContractInvocationOperation(
-        network: string,
+        server: string,
         keyStore: KeyStore,
         to: string,
         amount: number,
         fee: number,
         derivationPath: string,
-        storage_limit: string,
-        gas_limit: string,
-        parameters?: any
+        storageLimit: number,
+        gasLimit: number,
+        parameters?: string
     ) {
-        const blockHead = await TezosNodeReader.getBlockHead(network);
-        const sourceAccount = await TezosNodeReader.getAccountForBlock(network, blockHead.hash, keyStore.publicKeyHash);
+        const blockHead = await TezosNodeReader.getBlockHead(server);
+        const sourceAccount = await TezosNodeReader.getAccountForBlock(server, blockHead.hash, keyStore.publicKeyHash);
 
         let transaction: TezosTypes.Operation = {
             destination: to,
             amount: amount.toString(),
-            storage_limit,
-            gas_limit,
+            storage_limit: storageLimit.toString(),
+            gas_limit: gasLimit.toString(),
             counter: (Number(sourceAccount.counter) + 1).toString(),
             fee: fee.toString(),
             source: keyStore.publicKeyHash,
             kind: "transaction"
         };
 
         if (!!parameters) {
-            (<TezosTypes.ContractInvocationOperation> transaction).parameters = parameters;
+            const michelineParams = TezosLanguageUtil.translateMichelsonToMicheline(parameters);
+            (<TezosTypes.ContractInvocationOperation> transaction).parameters = JSON.parse(michelineParams).script;
         }
 
-        const operations = await appendRevealOperation(network, keyStore, sourceAccount, [transaction]);
-        return sendOperation(network, operations, keyStore, derivationPath);
-    }
-
-    /**
-     * Indicates whether an account is implicit and empty. If true, transaction will burn 0.257tz.
-     * @param {string} network Which Tezos network to go against
-     * @param {KeyStore} keyStore Key pair along with public key hash
-     * @returns {Promise<boolean>} Result
-     */
-    export async function isImplicitAndEmpty(network: string, accountHash: string): Promise<boolean> {
-        const blockHead = await TezosNodeReader.getBlockHead(network);
-        const account = await TezosNodeReader.getAccountForBlock(network, blockHead.hash, accountHash);
-
-        const isImplicit = accountHash.toLowerCase().startsWith("tz");
-        const isEmpty = Number(account.balance) === 0
-
-        return (isImplicit && isEmpty)
-    }
-
-    /**
-     * Indicates whether a reveal operation has already been done for a given account.
-     *
-     * @param {string} network Which Tezos network to go against
-     * @param {KeyStore} keyStore Key pair along with public key hash
-     * @returns {Promise<boolean>} Result
-     */
-    export async function isManagerKeyRevealedForAccount(network: string, keyStore: KeyStore): Promise<boolean> {
```

Comment:
This is a function used in wallet/thunks in Tezori. Deleting it causes problems.

Reply:
Yes, this is now in https://github.com/Cryptonomic/ConseilJS/blob/feature/governance/src/chain/tezos/TezosNodeReader.ts#L101